app/views/referee_interface/reference/feedback.html.erb

```diff
@@ -7,11 +7,10 @@
       <h1 class="govuk-heading-xl">Give a teacher training reference for <%= @application.full_name %></h1>
 
       <p class="govuk-body"><%= @application.full_name %> applied to the following courses:</p>
+
       <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-6">
         <% @application.application_choices.each do |application_choice| %>
-          <li>
-            <%= application_choice.course.name %> - <%= application_choice.site.name %>
-          </li>
+          <li><%= application_choice.course_and_site %></li>
```

Comment:
If `course_and_site` this is used only here I'd keep it as it is now, so we don't add extra methods to the models.

Reply:
yeah, reasonable.